samples/rag-aisearch/csharp-ooproc/FilePrompt.cs

```diff
@@ -29,38 +32,23 @@ public class SemanticSearchRequest
     // REVIEW: There are several assumptions about how the Embeddings binding and the SemanticSearch bindings
     //         work together. We should consider creating a higher-level of abstraction for this.
     [Function("IngestFile")]
-    public static async Task<SemanticSearchOutputResponse> IngestFile(
+    public static async Task<HttpResponseData> IngestFile(
         [HttpTrigger(AuthorizationLevel.Function, "post")] HttpRequestData req,
-        [EmbeddingsInput("{FilePath}", InputType.FilePath, Model = "%EMBEDDING_MODEL_DEPLOYMENT_NAME%")] EmbeddingsContext embeddings)
+        [EmbeddingsStoreInput("{URL}", InputType.URL, "{Title}", "AISearchEndpoint", "openai-index", Model = "%EMBEDDING_MODEL_DEPLOYMENT_NAME%")] EmbeddingsContext embeddings)
```

Comment:
I'm confused about why this is an input binding instead of an output binding. It seems like we're using this to store data into an embeddings store, right? Shouldn't that be the job of an output binding?

Reply:
updated to output